packages/cosmwasm-launchpad/src/signingcosmwasmclient.ts

```diff
@@ -108,10 +108,8 @@ export interface InstantiateOptions {
    * created and before the instantiation message is executed by the contract.
    *
    * Only native tokens are supported.
-   *
-   * TODO: Rename to `funds` for consistency (https://github.com/cosmos/cosmjs/issues/806)
    */
-  readonly transferAmount?: readonly Coin[];
+  readonly funds?: readonly Coin[];
```

Comment:
While you're here, could you duplicate `interface InstantiateOptions` to both packages?